index.js

```diff
@@ -103,6 +126,9 @@ console.log('hello world')
  */
 
 // TODO: ADD CODE HERE
+stringToUpper = (str) => str.toUpperCase();
```

Comment:
You should use a variable keyword when declaring a new function (i.e. `const stringToUpper = (str) => {...}`)